ibis/backends/mysql/tests/conftest.py

```diff
@@ -48,6 +50,67 @@ def __init__(self, data_directory: Path) -> None:
             # mysql supports window operations after version 8
             self.__class__.supports_window_operations = True
 
+    @staticmethod
+    def load_data(data_dir: Path, script_dir: Path, **kwargs) -> None:
+        """Load testdata into a mysql backend.
+
+        Parameters
+        ----------
+        data_dir : Path
+            Location of testdata
+        script_dir : Path
+            Location of scripts defining schemas
+        """
+        user = (
+            MYSQL_USER
+            if not kwargs.get("username", None)
+            else kwargs["username"]
+        )
+        password = (
+            MYSQL_PASS
+            if not kwargs.get("password", None)
+            else kwargs["password"]
+        )
+        host = MYSQL_HOST if not kwargs.get("host", None) else kwargs["host"]
+        port = MYSQL_PORT if not kwargs.get("port", None) else kwargs["port"]
+        database = (
+            IBIS_TEST_MYSQL_DB
+            if not kwargs.get("database", None)
+            else kwargs["database"]
+        )
+
+        schema = (
+            open(script_dir / 'schema' / 'mysql.sql')
+            if not kwargs.get("schema", None)
+            else kwargs["schema"]
+        )
+        tables = (
+            list(TEST_TABLES.keys())
+            if not kwargs.get("tables", None)
+            else kwargs["tables"]
+        )
+
+        engine = init_database(
+            url=sa.engine.make_url(
+                f"mysql+pymysql://{user}:{password}@{host}:{port}?local_infile=1",  # noqa: E501
+            ),
+            database=database,
+            schema=schema,
+            isolation_level="AUTOCOMMIT",
+        )
+        with engine.connect() as con:
+            for table in tables:
+                con.execute(
+                    f"""\
```

Comment:
I like to use `textwrap.dedent()`  to preserve the tabulation of the python source code.